### ISSUE TITLE
interferes with some gestures and scrollViews on iOS

### ISSUE THREAD

**Issue by wookelvin** [negativetwelve/react-native-ux-cam#46]
On iOS, when implementing a scrollview, the scrollview actually stutters when UXCam is recording. 
It's very obvious and reduces the overall performance of an app.

When implemented with DrawerView from react-native-navigation, the drawer itself also no longer works correctly. (likely due to same stutter issue). 

I'm no longer implementing this due to the several performance issues this is creating.

**Comment by negativetwelve** [comment]
There's a pretty long discussion here: https://github.com/negativetwelve/react-native-ux-cam/issues/36

This is also an issue with the native library and not the react-native wrapper. Your best bet would be to contact support and talk to the engineering team at UXCam. Sorry I couldn't help more!

**Comment by negativetwelve** [comment]
Going to close this as it's an issue with the native library and not the react-native wrapper. You can definitely continue discussion via this issue if you'd like!

**Comment by richardgroves** [comment]
Hi @wookelvin 

We've seen this issue - it's direct in the SDK and not in the react wrapper.
It appears that something changed around iOS11.2 that causes extra time to be taken for the screen capture we do when the view is scrolling. There should be an update SDK (2.5.15) released for iOS today that has a BETA method 'StopRecordingScrollingOnStutterOS' that works around it by stopping screen recording when a view is scrolling on iOS11.2.

We are working on finding out exactly what has changed in the OS and how to return capture functionality to normal.

**Comment by kevinmcalear** [comment]
@negativetwelve it would be amazing if you could update this library to 2.5.15 and add the StopRecordingScrollingOnStutterOS method to this library. Thanks!

**Comment by kevinmcalear** [comment]
@richardgroves did you ever fix this?

**Comment by negativetwelve** [comment]
@kevinmcalear I can do that later this week, but if you'd like to take a stab at it, I would appreciate it 😄

**Comment by richardgroves** [comment]
@kevinmcalear We are still trying to work out what the root problem is - from test code we have it appears to be definitely related to an iOS update. I have a radar open with Apple about it. My current theory is that it is related to the Meltdown/Spectre fixes that went into one of the 11.2.X iOS versions.

The problem is noticeably worse on iPhoneX, but is present on other devices.

If you are updating the React-wrapper please try and use the 2.5.16 SDK as that has some issues found in 2.5.15 fixed.

**Comment by kevinmcalear** [comment]
@negativetwelve I upgraded to 2.5.16 (as @richardgroves suggested, thanks!) and implemented the `StopRecordingScrollingOnStutterOS` method. This does fix the stuttering of the ScrollView. This, however, does not fix the core issue for me though, which is documented in [this issue](https://github.com/alexbrillant/react-native-deck-swiper/issues/122).

For reference, I just upgraded the CocoaPod to 2.5.16 and modifed `startWithKey` as shown below:

```
RCT_EXPORT_METHOD(startWithKey:(NSString *)key) {
  [UXCam startWithKey:key];
  [UXCam StopRecordingScrollingOnStutterOS:true];
}
```
